finished/src/main/java/com/example/android/wearable/composeforwearos/MainActivity.kt

```diff
@@ -63,61 +58,56 @@ class MainActivity : ComponentActivity() {
     }
 }
 
+@OptIn(ExperimentalHorologistApi::class)
 @Composable
 fun WearApp() {
     WearAppTheme {
-        // TODO: Swap to ScalingLazyListState
-        val listState = rememberScalingLazyListState()
+        /*
+         * [Horologist] AppScaffold adds a TimeText by default that can be override by the
+         * ScreenScaffold. It ensures that TimeText animates correctly when navigating between
+         * screens.
+         * */
+        AppScaffold {
+            // TODO: Swap to ScalingLazyListState
+        /*
+         * Specifying the types of items that appear at the start and end of the list ensures that the
+         * appropriate padding is used.
+         */
+            val listState = rememberResponsiveColumnState(
+                contentPadding = ScalingLazyColumnDefaults.padding(
+                    first = ItemType.Text,
+                    last = ItemType.SingleButton,
```

Comment:
I think you have a Chip?

Reply:
yep and also the first one is a button, fixed